.github/workflows/test-package.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - run: dart analyze --fatal-infos
         if: matrix.sdk == 'dev' && steps.install.outcome == 'success'
       - run: dart analyze
-        if: matrix.sdk == 'stable' && steps.install.output == 'success'
+        if: matrix.sdk == '2.12.0' && steps.install.output == 'success'
```

Comment:
this is what you meant, right @natebosch ?

Reply:
yes